README.md

```diff
@@ -45,7 +45,7 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
Let's try to update our README a little bit, in such a way that a non-technical person could read through the README and start the app.

from your README 👇 
![image](https://user-images.githubusercontent.com/31897434/92537295-adb81e00-f233-11ea-8962-b0715757ab26.png)

it would be more helpful this way 👇 
![image](https://user-images.githubusercontent.com/31897434/92537386-e5bf6100-f233-11ea-89a2-9729e2bd75fd.png)